src/azureFunction/azureFunctionUtils.ts

```diff
@@ -0,0 +1,90 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import * as fse from 'fs-extra';
+import * as path from 'path';
+import * as vscode from 'vscode';
+import * as constants from '../constants/constants';
+import * as LocalizedConstants from '../constants/localizedConstants';
+// https://github.com/microsoft/vscode-azuretools/blob/main/ui/api.d.ts
```

Comment:
This isn't the correct link - this typings file is from https://github.com/microsoft/vscode-azurefunctions/blob/main/src/vscode-azurefunctions.api.d.ts